ui/asset_table.js

```diff
@@ -63,7 +63,13 @@ AssetTable.prototype.populate = function (table_data, draw_cb) {
     let init_obj = {
         "data": data,
         "columns": [
-            {"data": first_column_name, "title": string_capitalize(first_column_name)},
+            {
+                "title": string_capitalize(first_column_name),
+                "data": first_column_name,
+                "render": function (data, type, row) {
+                    return format_asset_title_for_ui(row['asset']);
```

Comment:
This won't always work. Some asset tables use 'currency' as the key of the asset (I should fix that) ... but also in some tables (like the per account balances) the first column is not an asset. In the per accounts table it's an account. All of these should be handled separately here.